score/ipython/hud_housing_etl.ipynb

```diff
@@ -0,0 +1,274 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c21b63a3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
```

Comment:
Nothing just a comment, but all this source-as-array seems less than ideal, but I guess that's .ipynb?

Reply:
Yeah- this bit was from having merged with Lucas's changes while developing this branch. Agreed that notebooks outside of being rendered look messy in source; notably with Jorge's latest changes we've got more pure python fns now